bin/accessibility-automation/helper.js

```diff
@@ -0,0 +1,219 @@
+const logger = require("../helpers/logger").winstonLogger;
+const { API_URL } = require('./constants');
+const utils = require('../helpers/utils');
+const fs = require('fs');
+const path = require('path');
+const request = require('request');
+const os = require('os');
+const glob = require('glob');
+const helper = require('../helpers/helper');
+
+exports.checkAccessibilityPlatform = (user_config) => {
```

Comment:
wrap in try catch, return false by default

Reply:
done